main.js

```diff
@@ -0,0 +1,166 @@
+    htmlSkeletonToInsert += `<div class="card">
+     <div class="card-img">
+       <img
+         src="${data.image}"
+         alt="Tonic-project-image"
+         title="Tonic-project"
+       />
+     </div>
+     <div class="card-content facebook-card">
+       <h3 class="card-title">${data.name}</h3>
+       <ul class="card-info">
+         <li class="bold-info">${data.tag.sector}</li>
+         <li class="info bullet"><span></span>${data.tag.type}</li>
+         <li class="info bullet"><span></span>${data.tag.year}</li>
+       </ul>
+       <p class="card-para">
+         ${data.description}
+       </p>
+       <ul class="card-lang">
+      ${techList}
+      </ul>
+       <button class="card-btn" data-value="${index}">See Project</button>
+     </div>
+   </div>
+     `;
+  }
+  projectData.map(insertData);
+  document.getElementById('portflio').innerHTML = htmlSkeletonToInsert;
+}());
+
+function closeProjectPopup() {
+  document.getElementsByClassName('popup-section')[0].style.display = 'none';
+}
+
+const triggerElem = document.getElementsByClassName('card-btn');
+function openProjectPopup(e) {
+  e.preventDefault();
+  const targetIndex = e.target.attributes[1].value;
+  const techList = insertTechList(projectData[targetIndex]);
+
+  const elemToInsert = `
+   <h3 class="title work-title">
+     ${projectData[targetIndex].name}
+     <img  class="clossing-project"src="images/Iconx.png" alt="">
+     </h3>
+    
+     <div class="work-info">
+     <ul class="work-tag-popup">
+         <li class="first-li">CANOPY</li>
+        <li class="second-li">
+            <img src="./images/counter.png">
+            Back End Dev
+        </li>
+         <li class="second-li">
+            <img src="./images/counter.png">
+            2015
+         </li>
+     </ul>
+     </div>
+     <img src="${projectData[targetIndex].image}" alt="" class="popup-image mobile-pop-img">
+     <div class="work-tag-button-wrapper  work-mobile-wrapper">
+     <p class="work-description work-description-mobile">
+     ${projectData[targetIndex].details}
+     </p>
+     <div class="work-tag-wrapper">
+     <div class="work-tag">
+     <ul class="card-lang">
+     ${techList}
+     </ul>
+     </div>
+     <div class="see-container">
+     <a href="https://github.com/MariamOsman11/-Portfolio-.git" id="popup-link" class="see-live" target="_blank">
+     <span>seeLive</span>
+     <img src="./images/live.png" class="see-image">
+     </a>
+     <a href="https://github.com/MariamOsman11/-Portfolio-.git" id="popup-link" class="see-live">
+     <span>seeSource</span>
+     <img src="./images/github.png" class="see-image">
+     </a>
+     </div>
+     </div>`;
+
+  document.getElementsByClassName('window-wrapper')[0].innerHTML = elemToInsert;
+  document.getElementsByClassName('popup-section')[0].style.display = 'block';
+  const closeElemPopup = document.getElementsByClassName('clossing-project')[0];
+  closeElemPopup.addEventListener('click', closeProjectPopup);
+}
```

Comment:
- [ ] Kindly note that your design overflows on mobile screen which can affect the UX, this is probably due to the unwrapped flex of the programming languages and frameworks (bootstrap and the like).
![mobile (15)](https://github.com/siddghosh108/Portfolio-Popup-Window/assets/41163571/f90188cb-3294-460e-aa30-845437a1dbae)